Newsletters: add 239 (2023-02-22)

harding opened the pull request:
- [x] Lede, releases/RCs, topic links @harding 
- [x] Bitcoin SE section @bitschmidty 
- [x] Bitcoin Core 24149 @Xekyo

paulkania reviewed (commented):
1 minor typo, a few nits. Hope I'm helping.

vostrnad commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,212 @@
+    [Newsletter #226][news226 jam]), this is part of a proposal to
+    mitigate [channel jamming attacks][topic channel jamming attacks].
+
+- **Proposed BIP for Codex32 seed encryption scheme:** Russell O'Connor
+  and Andrew Poelstra (using anagrams of their names) proposed a BIP for
+  a new scheme backing up and restoring [BIP32][] seeds.  Similar to
```
Missing link to the mailing list post.

bitschmidty replied:
https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-February/021469.html

LarryRuane commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+  it'll be able to forward payments without failure.  If it does fail to
+  forward a payment, the spender may choose not to use that node for
+  future payments for a very long time---a much longer duration than the
+  spender may use for nodes which did not advertise high availability.
```
```suggestion
  spender may use for nodes that did not advertise high availability.
```

LarryRuane commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+- **Feedback requested on LN good neighbor scoring:** Carla Kirk-Cohen
+  and Clara Shikhelman [posted][ckc-cs reputation] to the Lightning-Dev
+  mailing list to request feedback on the recommend parameters for how a
+  node judges whether its channel counterparties are a good source of
```
```suggestion
  node should judge whether its channel counterparties are a good source of
```

LarryRuane commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+  mailing list to request feedback on the recommend parameters for how a
+  node judges whether its channel counterparties are a good source of
+  forwarded payments.  They suggest several criteria to judge by and
+  recommend default parameters for each criteria, but are seeking
```
```suggestion
  recommend default parameters for each criterion but are seeking
```

LarryRuane commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+  and Andrew Poelstra (using anagrams of their names) proposed a BIP for
+  a new scheme backing up and restoring [BIP32][] seeds.  Similar to
+  [SLIP39][], it optionally allows creating several shares using
+  [Shamir's Secret Sharing Scheme][] (SSSS) that requires a configurable
```
```suggestion
  [Shamir's Secret Sharing Scheme][] (SSSS), which requires a configurable
```

LarryRuane commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+    keep information safe and secure by distributing it across multiple
+    places or people.  With Codex32, users can verify the integrity of
+    each share individually on a regular basis using just paper, pen,
+    some printed materials, and a few minutes of time.
```
That is very cool, and I like the enthusiasm of the preceding paragraph!

LarryRuane commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+
+    Discussion on the mailing list mainly focused on the differences
+    between Codex32 and SLIP39, which has been used in production for a
+    couple years now.  We recommend anyone interested in Codex32
```
```suggestion
    couple of years now.  We recommend anyone interested in Codex32
```

LarryRuane commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+  transaction compression scheme.
+
+- [How does one become a DNS seed for Bitcoin Core?]({{bse}}116931)
+  User Paro explains requirements for someone wanting to become a [DNS
```
```suggestion
  User Paro explains the requirements for someone wanting to become a [DNS
```

LarryRuane reviewed (commented):
Looks pretty good, I verified the SE numbers and the Bitcoin Core PR numbers.

bitschmidty commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
```
Sure @LarryRuane do you want to take a crack at the writeup?

LarryRuane replied:
> Sure @LarryRuane do you want to take a crack at the writeup?

Pushed a separate commit, ad1e6119, feel free to drop or squash.

Xekyo commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,265 @@
+  management: to determine which inputs need to be added to a
+  transaction, the amount of the fee needs to be known, but to determine
+  the amount of the fee, the size of the transaction needs to be known.
+  The function can provide a reasonable size estimate without actually
+  having to construct a candidate transaction.
```
Mh, they’re part of the Lucky 10,000 today. I just wrote them a few lines about using the _effective values_ of UTXOs during coin selection.

LarryRuane commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,212 @@
+- [Bitcoin Core #24149][] Signing support for Miniscript Descriptors FIXME:Xekyo
+
+- [Bitcoin Core #25344][] updates the `bumpfee` and `psbtbumpfee` RPCs
+  for using [Replace By Fee][topic rbf] (RBF) to allow specifying a
```
Ah, I see what you mean, @Xekyo, without the "for" it would seem that those RPCs could use means _other than_ RBF to do something ... which is incorrect; they always / can only use RBF. I still think it's a little unclear; maybe something like
```suggestion
  , which use [Replace By Fee][topic rbf] (RBF), to allow the specification of a
```

bitschmidty commented:
ACK the content from [ad1e611](https://github.com/bitcoinops/bitcoinops.github.io/pull/1018/commits/ad1e611967a3b0105675ffdc37c970c85b615654)
Thanks @LarryRuane !

harding commented:
Pushed edits for all feedback (thanks @paulkania, @bitschmidty, @LarryRuane , @vostrnad, and @bitschmidty !).  Added lede, updated releases/RCs, and added topic link.  Also extended the description of Eclair 2596 with additional information thanks to the always amazing @t-bast.  Reviewed the contributions by @bitschmidty, @Xekyo, and @LarryRuane (wonderful, thanks!).

I think this is going to be a good newsletter, thanks everyone!

jirijakes commented on _posts/en/newsletters/2023-02-22-newsletter.md:
```diff
@@ -0,0 +1,291 @@
+    [Newsletter #226][news226 jam]), this is part of a proposal to
+    mitigate [channel jamming attacks][topic channel jamming attacks].
+
+- **Proposed BIP for Codex32 seed encryption scheme:** Russell O'Connor
```
```suggestion
- **Proposed BIP for Codex32 seed encoding scheme:** Russell O'Connor
```

If I understand correctly, it does not encrypt anything.

bitschmidty replied:
Took this suggestion

bitschmidty commented:
Squashed and merged, thank you @harding @LarryRuane @Xekyo for authoring this week and @jirijakes @vostrnad @LarryRuane @paulkania @Xekyo for the reviews!
